Snake/BusinessLayer/BusinessLayer.csproj.user

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
You should ignore the .user setting for proj file, commit this mean everyone gets the same settings, I may want different settings to other people using the project, for example here I dont like seeing all files.